lib/meta_tags/text_normalizer.rb

```diff
@@ -18,7 +18,8 @@ def self.normalize_title(title)
     #
     def self.normalize_description(description)
       return '' if description.blank?
-      helpers.truncate(cleanup_string(description), :length => 200)
+      helpers.truncate(cleanup_string(description),
+                       :length => MetaTags.truncate_description_at_length)
```

Comment:
Older hash syntax is prevalent in the rest of the gem, so would prefer to retain it.